diff --git a/node_modules/react-query/types/react/QueryClientProvider.d.ts b/node_modules/react-query/types/react/QueryClientProvider.d.ts
index 65e1214..ea9fbf0 100644
--- a/node_modules/react-query/types/react/QueryClientProvider.d.ts
+++ b/node_modules/react-query/types/react/QueryClientProvider.d.ts
@@ -10,4 +10,4 @@ export interface QueryClientProviderProps {
     client: QueryClient;
     contextSharing?: boolean;
 }
-export declare const QueryClientProvider: React.FC<QueryClientProviderProps>;
+export declare const QueryClientProvider: React.FC<React.PropsWithChildren<QueryClientProviderProps>>;
